packages/browser/src/tester/ajs-tester.ts

```diff
@@ -12,11 +12,13 @@ function makeStub(page: playwright.Page) {
       ...args: Parameters<Analytics['register']>
     ): Promise<SerializedContext> {
       return await page.evaluate((innerArgs) => {
-        // @ts-ignore
-        return window.analytics
-          .register(...innerArgs)
-          .then((ctx) => ctx.toJSON())
-        // @ts-ignore
+        return (
```

Comment:
It's actually fine to do declare global  in test code, since it doesn't get included in the actual published library.